_haikus/Note_to_self.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Note to self
+author: Abubakar
+---
+
+
+In life being truthful
```

Comment:
This line should have 5 syllables and should end with two spaces `  `

Maybe you can try

```
In life, being true  
```